src/g_all_guis.c

```diff
@@ -517,9 +525,17 @@ void iemgui_label_font(void *x, t_iemgui *iemgui, t_symbol *s, int ac, t_atom *a
         f = 4;
     iemgui->x_fontsize = f;
     if(glist_isvisible(iemgui->x_glist))
-        sys_vgui(".x%lx.c itemconfigure %lxLABEL -font {{%s} -%d %s}\n",
-                 glist_getcanvas(iemgui->x_glist), x, iemgui->x_font,
-                 iemgui->x_fontsize*zoom, sys_fontweight);
+    {
+        char tag[128];
+        t_atom fontatoms[3];
+        sprintf(tag, "%lxLABEL", x);
+        SETSYMBOL(fontatoms+0, gensym(iemgui->x_font));
+        SETFLOAT (fontatoms+1, -iemgui->x_fontsize*zoom);
```

Comment:
i think this line should be `SETFLOAT (fontatoms+1, iemgui->x_fontsize*zoom);`

there is a "minus" sign `-iemgui->...` I think is a typo

Reply:
It's indeed intentional, as negative numbers are Tcl/Tk's way to express absolute (non-scalable) fontsizes.
the original code added the `-` in the format-string...